File: clsObj/linklst/linkedlist.java
public class linkedlist {

    private class Node {
        int data = 0;
        Node next = null;

        Node (int data) {
            this.data = data;
        }
    }

    private Node head = null;    
    private Node tail = null;
    private int size = 10;

    public int getSize() {
        return size;
    }

    public boolean isSizeEmpty() {
        return size == 0;
    }

    public void firstNode (int data) {
        Node node = new Node(data);
        addFirstNode(node);
    }
    public void addFirstNode(Node node) {
        if (isSizeEmpty()) {
            this.head = this.tail = node;
        }
        else {
            node.next = this.head;
            this.head = node;
        }
    }

}

File: clsObj/linklst/linkedlist.py
class LinkedList:

    class Node:
        def __init__(self, data):
            self.data = data
            self.next = None

    def __init__(self):
        self.head = None
        self.tail = None
        self.size = 10

    def get_size(self):
        return self.size

    def is_size_empty(self):
        return self.size == 0

    def first_node(self, data):
        node = self.Node(data)
        self.add_first_node(node)

    def add_first_node(self, node):
        if self.is_size_empty():
            self.head = self.tail = node
        else:
            node.next = self.head
            self.head = node